UnitsNet.Tests/CustomCode/PorousMediumPermeabilityTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by \generate-code.bat.
+//
+//     Changes to this file will be lost when the code is regenerated.
+//     The build server regenerates the code before each build and a pre-build
+//     step will regenerate the code on each local build.
+//
+//     See https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit for how to add or edit units.
+//
+//     Add CustomCode\Quantities\MyQuantity.extra.cs files to add code to generated quantities.
+//     Add UnitDefinitions\MyQuantity.json and run generate-code.bat to generate new units or quantities.
+//
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+// Licensed under MIT No Attribution, see LICENSE file at the root.
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using System;
+using System.Globalization;
+using UnitsNet.Units;
+using Xunit;
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class PorousMediumPermeabilityTests : PorousMediumPermeabilityTestsBase
+    {
+        protected override bool SupportsSIUnitSystem => true;
+
+        protected override double DarcysInOneSquareMeter => 0.10132499658e13;
+
+        protected override double MicrodarcysInOneSquareMeter => 0.10132499658e19;
+
+        protected override double MillidarcysInOneSquareMeter => 0.10132499658e16;
+
+        protected override double SquareCentimetersInOneSquareMeter => 10000;
+
+        protected override double SquareMetersInOneSquareMeter => 1;
+
+        [Theory]
+        [InlineData("2 D", 2, PorousMediumPermeabilityUnit.Darcy)]
+        [InlineData("2 mD", 2, PorousMediumPermeabilityUnit.Millidarcy)]
+        [InlineData("2 µD", 2, PorousMediumPermeabilityUnit.Microdarcy)]
+        [InlineData("2 m²", 2, PorousMediumPermeabilityUnit.SquareMeter)]
+        [InlineData("2 cm²", 2, PorousMediumPermeabilityUnit.SquareCentimeter)]
+        public void Parse(string input, double value, PorousMediumPermeabilityUnit unit)
```

Comment:
I thought these tests would be redundant, but I see we have forgot to generate tests for parse on each individual quantity. I will fix outside this PR.

Reply:
Added tests in https://github.com/angularsen/UnitsNet/pull/1086